cf_xarray/accessor.py

```diff
@@ -308,7 +338,7 @@ def _get_measure(da: Union[DataArray, Dataset], key: str) -> List[str]:
     "dims_or_levels": (_get_axis_coord,),  # reset_index
     "window": (_get_axis_coord,),  # rolling_exp
     "coord": (_get_axis_coord_single,),  # differentiate, integrate
-    "group": (_get_axis_coord_single,),
+    "group": (_get_axis_coord_single, _get_axis_coord_time_accessor),
```

Comment:
missed a step ;)

Reply:
No `,` after `_get_axis_coord_time_accessor` ?

Reply:
I added `_get_axis_coord_time_accessor`. It was missing. I think your branch was a little messed up after the `pull` bot did its thing.